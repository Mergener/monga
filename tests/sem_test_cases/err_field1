type Vec2 = {
	x : float;
	y : float;
};

function main() {
	var v : Vec2;

	v.x = 5 as float;
	v.y = 3 as float;
	v.z = 2 as float;
}