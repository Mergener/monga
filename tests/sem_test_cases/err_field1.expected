[Error] At line 11, column 5: Specified field 'z' does not exist in value of type 'Vec2'.
Semantic errors were caught during analysis.
